api/v2/utils.go

```diff
@@ -255,13 +255,21 @@ func (api *API) validateHoldTime(username, holdTime string) (int64, error) {
 	return holdTimeInt, nil
```

Comment:
It doesn't let me comments on the line, but there is an error message that needs to be updated 5 lines above.

Reply:
nice catch